hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-config-scm4

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+OZONE-SITE.XML_ozone.scm.nodes.scmservice=scm1,scm2,scm3,scm4
+OZONE-SITE.XML_ozone.scm.address.scmservice.scm4=scm4.org
```

Comment:
Nit: no newline at and of file, there was an effort to have newlines at the end of all files, it is not enforce for non-Java files, but it would be nice to keep the convention.